docs/source/reference/api-server/api-server-upgrade.rst

```diff
@@ -0,0 +1,240 @@
+.. _sky-api-server-upgrade:
+
+Upgrading SkyPilot API Server
+=============================
+
+This page provides an overview of the steps you should follow to upgrade a remote SkyPilot API server:
+
+* :ref:`sky-api-server-helm-upgrade`
+* :ref:`sky-api-server-vm-upgrade`
+
+.. _sky-api-server-helm-upgrade:
+
+Upgrade API server deployed with Helm
+-----------------------------------------
+
+Here we introduce the steps for upgrading a remote API server deployed with :ref:`Helm deployement <sky-api-server-deploy>`.
+
+.. note::
+
+    Upgrading the API server introduces downtime. We recommend to schedule the upgrade during a **maintenance window**: cordon & drain the old API server and upgrade.
+
+Step 1: Prepare an upgrade
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Find the version to use in SkyPilot `nightly build <https://pypi.org/project/skypilot-nightly/#history>`_.
+2. Update SkyPilot helm repository to the latest version:
+
+.. code-block:: bash
+
+    helm repo update skypilot
+
+3. Prepare variables for the upgrade, the ``NAMESPACE`` and ``RELEASE_NAME`` should be the same as the previous deployment step:
```

Comment:
```suggestion
3. Prepare versioning environment variables.  ``NAMESPACE`` and ``RELEASE_NAME`` should be set to the currently installed namespace and release:
```
By the way, what does "release name" mean?

Reply:
A term in helm, represents a deployed instance. Here we referring the ``RELEASE_NAME`` user used in deployment stesp https://docs.skypilot.co/en/latest/reference/api-server/api-server-admin-deploy.html#step-3-deploy-the-api-server-helm-chart